pom.xml

```diff
@@ -479,5 +569,97 @@
         <quarkus.package.type>native</quarkus.package.type>
       </properties>
     </profile>
+
+    <profile>
+      <id>h2</id>
+      <activation>
+        <property>
+          <name>maven.jdbc.driver.class</name>
+          <value>org.h2.Driver</value>
+        </property>
+      </activation>
+      <properties>
+        <maven.jdbc.db-kind>h2</maven.jdbc.db-kind>
+      </properties>
+    </profile>
+
+    <profile>
+      <id>postgresql</id>
+      <activation>
+        <property>
+          <name>maven.jdbc.driver.class</name>
+          <value>org.postgresql.Driver</value>
+        </property>
+      </activation>
+      <properties>
+        <maven.jdbc.db-kind>postgresql</maven.jdbc.db-kind>
+      </properties>
+    </profile>
+
+    <profile>
+      <id>db2</id>
+      <activation>
+        <property>
+          <name>maven.jdbc.driver.class</name>
+          <value>com.ibm.db2.jcc.DB2Driver</value>
+        </property>
+      </activation>
+      <properties>
+        <maven.jdbc.db-kind>db2</maven.jdbc.db-kind>
+      </properties>
+    </profile>
+
+    <profile>
+      <id>mariadb</id>
+      <activation>
+        <property>
+          <name>maven.jdbc.driver.class</name>
+          <value>org.mariadb.jdbc.Driver</value>
+        </property>
+      </activation>
+      <properties>
+        <maven.jdbc.db-kind>mariadb</maven.jdbc.db-kind>
+        <maven.pim.flyway.location>db/migration/mariadb</maven.pim.flyway.location>
```

Comment:
This profile is the only one that contains "maven.pim.flyway.location", shouldn't it be the same for all?

Reply:
Thanks for that! it's not needed. Added to the next iteration